### ISSUE TITLE
ol is not defined

### ISSUE THREAD

**Issue by 9geeks** [jonataswalker/ol3-contextmenu#97]
When I using this package in npm, I have an error:

```
Uncaught ReferenceError: ol is not defined
    at ol3-contextmenu.js:8
    at e (ol3-contextmenu.js:8)
    at Object.<anonymous> (ol3-contextmenu.js:8)
    at __webpack_require__ (bootstrap 3ed0ba6…:553)
    at fn (bootstrap 3ed0ba6…:84)
    at Object.<anonymous> (map.js:76252)
    at Object.<anonymous> (map.js:76253)
    at __webpack_require__ (bootstrap 3ed0ba6…:553)
    at fn (bootstrap 3ed0ba6…:84)
```

I think we should `import ol from 'openlayers';` in source files

**Comment by jonataswalker** [comment]
`openlayers` is a declared dependency in `package.json`. It should be installed along `ol3-contextmenu`.

How to reproduce your issue?

**Comment by 9geeks** [comment]
I am using openlayers3 with React.
I also add `ol3-contextmenu` and `openlayers` in package.json.
When I add import command to src/js/base.js then rebuild build `make build` it seems okay.
```
import ol from 'openlayers';
```

But I try to add context menu like your example, it have errors..
It seem this library is out of date, I am using openlayer v3.20.0.

Now I am making menu by React.

**Comment by jonataswalker** [comment]
Hang tight, I'll release a new version.

**Comment by 9geeks** [comment]
Thanks!!!

**Comment by jonataswalker** [comment]
@9geeks 

Published npm v2.4.0.

**Comment by 9geeks** [comment]
Thanks for your support ٩(^‿^)۶
